app/features/build_runner/build_runner.rb

```diff
@@ -170,17 +170,33 @@ def checkout_sha(&completion_block)
 
     def pre_run_action(&completion_block)
       logger.debug("Running pre_run_action in checkout_sha")
+
       checkout_sha do |checkout_success|
         if checkout_success
-          setup_build_specific_environment_variables
+          if setup_tooling_environment # see comment for `#setup_tooling_environment` method
```

Comment:
Could we join the two conditions `checkout_success` and `setup_tooling_environment` please?

i.e.,

```ruby
if checkout_success && setup_tooling_environment
  setup_build_specific_environment_variables
  completion_block.call(checkout_success)
else
  # ...
end
```

Reply:
Yep, will do, thanks